### ISSUE TITLE
Failed to start conversation: no conversation ID returned.

### ISSUE THREAD

**Issue by santiagodoldan** [OfficeDev/BotBuilder-MicrosoftTeams#84]
Hello guys, I'm getting this error when trying to start a conversation with some users, I'd like to know if this is something ok because some user roles can't interact with bots or if there's something I can do to start a conversation with them.

These are the lines that throw the error https://github.com/Microsoft/BotBuilder/blob/master/Node/core/src/bots/ChatConnector.ts#L314-L321

Thanks :)

**Comment by billbliss** [comment]
We need more information to answer your question.

What's **your** source code that's throwing the error?

**Comment by santiagodoldan** [comment]
```      
      this.connector.startConversation({
        ...address,
        channelId: "msteams",
      } as IAddress, (err, newAddress: ISuttnaAddress) => {
        ...
      })
```

Not sure if you mean that

**Comment by billbliss** [comment]
I can’t quite tell what the problem is. Are you saying that for some values of “...address” it’s working and for some it’s failing?

**Comment by santiagodoldan** [comment]
@billbliss Yes, I'm pretty sure it is because our bot can't start direct conversations with some users.

Saying that, this is what we're doing, we get all the users from a channel and then start a conversation with each user, over the past few weeks, we got a bunch of errors saying `Failed to start conversation: no conversation ID returned.`, I guess that it might be related to a specific user type or a role that does not have the proper privileges to have direct conversations with a bot, I'm just guessing because the `fetchMembers` only includes the id+name+email.

**Comment by billbliss** [comment]
There are no restrictions on bot conversations by user role. We should investigate. How can we find this in our logs - should we look for the Suttna bot and the "Failed to start conversation" error? What's your app id?

**Comment by santiagodoldan** [comment]
Is it ok to send some information to billbl@microsoft.com?

**Comment by santiagodoldan** [comment]
@billbliss I finally was able to reproduce this 😌 , I've created a new tenant, then invited a user from that new tenant as a guest, tried to start a direct conversation and got the "Failed to start conversation: no conversation ID returned."

Then I was able to fix it too, looks like if the guest user explicitly talks with our bot, that gives them privileges to talk with it.

Is this an issue or is the expected behavior? because if it is we must document it in our web or somewhere.

**Comment by bilby91** [comment]
This is kind of related to #88

**Comment by santiagodoldan** [comment]
@billbliss ping :)
